File: algorithms/sorting/selection-sort/python/selection-sort.py
def selection_sort(aList):
    for i in range(len(aList)):
        lowNumIndex = i
        for j in range(i+1, len(aList)):
            if aList[j] < aList[lowNumIndex]:
                lowNumIndex = j

        if aList[i] > aList[lowNumIndex]:
            tmp = aList[i]
            aList[i] = aList[lowNumIndex]
            aList[lowNumIndex] = tmp

    return aList

myList = [5, 7, 9, 4, 3, 8, 2]
print(selection_sort(myList))

File: algorithms/sorting/selection-sort/php/selection-sort.php
<?php

function do_sorting(array $array){
    for($i=0; $i<count($array); $i++){
        $lowNumIndex = $i;
        for($j=$i+1; $j<count($array); $j++){
            if($array[$j] < $array[$lowNumIndex]){
                $lowNumIndex = $j;
            }
        }
        if($array[$i] > $array[$lowNumIndex]){
            $tmp = $array[$i];
            $array[$i] = $array[$lowNumIndex];
            $array[$lowNumIndex] = $tmp;
        }
    }
    return $array;
}

$array = [5, 7, 9, 4, 3, 8, 2];
var_dump(do_sorting($array));